### PR TITLE
add button to email user

### PR DESCRIPTION
**Pull request by raniarusly**
image

### CODE REVIEW

File: src/main/webapp/styles/user-directory.css [hunk 1]
```diff
@@ -17,3 +16,14 @@
...
+
+.button {    
+  background: #38afac;
+  border: none;
+  color: white !important;
+  width: 150px;
+}
+
+.button:hover {    
+  background: #9afaf7;
```
Comment (lornaarmstrong) [inline src/main/webapp/styles/user-directory.css]: This colour makes the text unreadable so should probably be changed.

File: src/main/webapp/scripts/user-directory.js [hunk 2]
```diff
@@ -18,27 +18,32 @@ function generateUserCards() {
     {
       name: "User name 1",
       department: "Department1",
-      profilepic: "/images/avatar.jpg"
+      profilepic: "/images/avatar.jpg",
+      email: "useremail1@google.com"
```
Comment (MostafaGabriel) [inline src/main/webapp/scripts/user-directory.js]: Please use example domains (e.g. example.com) for the fake users

**Review by MostafaGabriel — APPROVED** [review]
Thanks Rania!

File: src/main/webapp/scripts/user-directory.js [hunk 3]
```diff
@@ -49,7 +54,9 @@ function generateUserCards() {
     const profilepicHTML = "<img class=\"user-img\" src=\"" + user.profilepic + "\"></img>";
     const nameHTML = "<h3>" + user.name + "</h3>";
     const departmentHTML = "<p>" + user.department + "</p>";
-    userCardElement.innerHTML = profilepicHTML + nameHTML + departmentHTML;
+    const emailTo = "<a href='mailto:" + user.email + "?subject=Enquiries for Student Recommendations'>message me</a>";
+    const button = "<div class='button' >" + emailTo + "</div>";
```
Comment (MostafaGabriel) [inline src/main/webapp/scripts/user-directory.js]: nit: emailButtonHTML for consistency?

**Review by MostafaGabriel — APPROVED** [review]

**Review by lornaarmstrong — APPROVED** [review]
LGTM!

**Pull request merged**